data/mods/classic_zombies/items/guns.json

```diff
@@ -88,5 +88,12 @@
     "name": { "str": "C6" },
     "copy-from": "m60",
     "description": "The Colt Canada C6 is a general-purpose machine gun developed to replace the .30-caliber C5 GPMG.  Heavy and difficult to handle fired from the shoulder, as most people aren't action-movie heroes."
+  },
+  {
+    "type": "GUN",
+    "id": "rossi_gallery",
+    "name": { "str": "Winchester 1890" },
```

Comment:
```suggestion
    "name": { "str": "Winchester 1890 rifle" },
```

Reply:
Why do I need "rifle" at the end?

Reply:
> Why do I need "rifle" at the end?

I was under the impression that, for weapons to be added into the main game, they require a descripter tag at the end of the description. I supposed that the same logic would be wise to be applied into classic zombies, unless the maintainers have objections. For reference, taken from the gun inclusion docs file:

> There are two rules for the default names of guns. That is, the names of guns
when the 'Gun Brand Names' option is not enabled. These rules are enforced by
script.

## Rule 1 (Descriptive Names): What is this?

This rule requires that the name of the gun clarifies what it is. This is meant
to prevent gun names that are obscure strings of letters and numbers.

Guns that follow this rule will have names like `Glock pistol`, `m4 carbine`,
or `UMP submachine gun`. Guns that don't will have names like `FS2000`,
`SIG P226`, `Ruger LCR .38`, or `Taurus Raging Bull`.

As of writing, names are considered to be acceptable if the gun name contains
any of these strings that describe that it is a gun and it's role. As with the
other rules, the most up-to-date info on what is acceptable is contained in the
[verification script](../tools/json_tools/gun_variant_validator.py).

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/74537 is trying to do it for with and without the option. I don't care, but if you want to be consistent with that you'd do it.